pybess/port.py

```diff
@@ -52,3 +52,7 @@ def get_port_stats(self):
 
     def get_link_status(self):
         return self.bess.get_link_status(self.name)
+
+    def set_port_config(self, **kwargs):
```

Comment:
Do we want to add this to `bessctl/commands.py` as well? (Otherwise it can only be invoked from scripts)